msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -374,54 +307,288 @@ Example for RSC in a chat
+
+* **Application context RSC permissions**: Users authorized to install an app in a specific scope have the right to grant any RSC permissions requested by the app in that specific scope. In addition to installing apps in a specific scope, the following constraints apply for the user to grant RSC permission to access data in app-only mode:
+  * In the case of a team, the user must be an owner of that team.
+  * In the case of a chat, the user must be a member of the chat.
+  * In the case of meetings, the user must be an organizer or presenter in the meeting.
+* **Delegated context RSC permissions**: Any user authorized to install an app in a specific scope has the right to grant any RSC permissions requested by the app in that specific scope at the time of installation. For example, if regular members are allowed to install an app inside a team, then they also have the authority to grant delegated RSC permission to the app in that specific team.
+
+If you're a global admin, you can review and grant consent to the apps that request permissions on behalf of all users in your organization. You do this so that users don't have to review and accept the permissions requested by the app when they open it. For more information, see [view app permissions and grant admin consent in Teams admin center](/microsoftteams/app-permissions-admin-center).
+
+## Add RSC permissions to your Teams application
+
+1. [Register your app with Microsoft identity platform using the Azure AD portal](#register-your-app-with-microsoft-identity-platform-using-the-azure-ad-portal).
+1. [Add application and delegated RSC permission to your app manifest](#update-your-teams-app-manifest).
+1. [Review your application permissions in the Azure AD portal](#review-your-application-permissions-in-the-azure-ad-portal).
+
+## Verify application RSC permission granted to your app
+
+### Prerequisite
+
+1. [Configure consent settings](#configure-consent-settings).
+    * Configure group owner consent settings for RSC in a team using the Azure AD portal.
+    * Configure chat owner consent settings for RSC in a chat using the Microsoft Graph APIs.
+
+To verify the application RSC permissions, follow these steps:
+
+1. [Install your app in Teams](#sideload-your-app-in-teams).
+1. [Obtain an access token from the identity platform](#obtain-an-access-token-from-the-microsoft-identity-platform).
+1. [Check the RSC permissions granted to a specific resource](#check-the-rsc-permissions-granted-to-a-specific-resource).
+    * Check your app for added RSC permissions in a team.
+    * Check your app for added RSC permissions in a chat.
+
+## Configure consent settings
+
+For application permissions, admins can enable or disable RSC permissions for all apps for the entire tenant using a toggle in Teams admin center. The tenant-level controls of app-only RSC permissions differ based on resource type.
```

Comment:
"For application permissions, admins can enable or disable RSC permissions for all apps for the entire tenant using a toggle in Teams admin center."
Why are we talking about this in the RSC doc? It gives the impression that there is a control for RSC permissions in TAC which is not true at present.

Reply:
Removed this sentence